testdata/local/scopes.json

```diff
@@ -1,4 +1,4 @@
-{ "_id" : "all_projects", "name" : "all projects", "type" : "project", "resources": ["logkeeper", "evergreen", "sys-perf", "spruce", "mongodb-mongo-master", "602d70a2b2373672ee493184", "mongodb-mongo-test", "performance"] }
+{ "_id" : "all_projects", "name" : "all projects", "type" : "project", "resources": ["logkeeper", "evergreen", "sys-perf", "spruce", "mongodb-mongo-master", "602d70a2b2373672ee493184", "mongodb-mongo-test", "performance", "parsley"] }
```

Comment:
I think you want to use the ID — when it's `parsley`, I can't see the project in the dropdowns as an admin, but when it's `602d70a2b2373672ee493189` I can
```suggestion
{ "_id" : "all_projects", "name" : "all projects", "type" : "project", "resources": ["logkeeper", "evergreen", "sys-perf", "spruce", "mongodb-mongo-master", "602d70a2b2373672ee493184", "mongodb-mongo-test", "performance", "602d70a2b2373672ee493189"] }
```

Reply:
Thanks!